examples/portfolio/src/pages/index.astro

```diff
@@ -5,6 +5,8 @@ import Button from '../components/Button/index.jsx';
 import Nav from '../components/Nav/index.jsx';
 import Footer from '../components/Footer/index.jsx';
 import PorfolioPreview from '../components/PortfolioPreview/index.jsx';
+import Hello from '../components/Hello.astro';
+import Another from '../components/Another.astro';
```

Comment:
oops, I thought I had checked these out, note to myself to remove these changes.